src/layout/Navbar/Navbar.scss

```diff
@@ -7,13 +7,15 @@
 }
 
 .navbar {
-	position: relative;
+	position: sticky;
+	inset-block-start: 0;
 	z-index: 10;
 	justify-content: space-between;
 	flex: 0 0 auto;
 	block-size: 58px;
 	padding: 12px 18px;
-	background-color: var(--fds-solid-background-base);
+	background-color: hsl(var(--mica-tint), var(--mica-tint-opacity));
+	backdrop-filter: blur(60px) saturate(150%);
```

Comment:
@yaira2 @TheOnlyTails What are your thoughts on reducing the blur radius? Right now, when scrolling over the images it flashes quite a lot, even when scrolling just a few pixels. I think something in the range of 20px would be better.

Reply:
I don't see flashes but could depend on the device, probably a good idea to reduce it.